riscv-asm.md

```diff
@@ -377,6 +377,49 @@ for PIC as seen by objdump:
 			4: R_RISCV_PCREL_LO12_I	.L0
 ```
 
+Load and Store Global
+---------------------
+
+The following pseudo instructions are available to load from and store to
+global objects:
+
+  * `l{b|h|w|d} <rd>, <symbol>`: load byte, half word, word or double word from global
+  * `s{b|h|w|d} <rd>, <symbol>, <rt>`: store byte, half word, word or double word to global
+  * `fl{h|w|d|q} <rd>, <symbol>, <rt>`: load half, float, double or quad precision from global
```

Comment:
Should the loads here have 3 arguments?

Reply:
For a load, rd is used as the temp register.  But that doesn't work for a store, so the temp register has to be specified explicitly.  And for a float load, we can't use an fp reg as an address temp, so it needs an explicit int temp reg also.

Reply:
Ah, so actually all the three instruction variants, including `s{b|h|w|d}`, use the third instruction as a temporary. Given I missed this, it might be useful to add a sentence just clarifying this (though I now see you can work it out from the code example shown).